solution-3.py

```diff
@@ -0,0 +1,12 @@
+bca_info = """BCA Course Information:Bachelor of Computer Applications (BCA) is a 3-year undergraduate program."""

```

Comment:
import statement? is it required or not?

Reply:
no it is not required.